### PR TITLE
proc_creation_win_invoke_sharefinder_discovery.yml

### PR DESCRIPTION
**Pull request by cyb3rjy0t**

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml:22]: The "Invoke-ShareFinder" command line is already cover https://github.com/search?q=repo%3ASigmaHQ%2Fsigma%20Invoke-ShareFinder&type=code

I will wait until #3959 is merged to add the base64 to 6385697e-9f1b-40bd-8817-f4a91f40508e and it's related rule.

**Review by nasbench — CHANGES_REQUESTED** [review]

File: rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml
Comment (cyb3rjy0t) [reply rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml:22]: Unable to see anything under "https://github.com/search?q=repo%3ASigmaHQ%2Fsigma%20Invoke-ShareFinder&type=code"

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml:22]: https://github.com/SigmaHQ/sigma/blob/1be13b3ea5fff5d5d2d846b0b7017c0453b0b1d4/rules/windows/process_creation/proc_creation_win_susp_sharpview.yml#L153

Comment (frack113) [reply rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml:22]: As it is a duplicate detection ,  I remove to no block the whole PR

**Review by frack113 — APPROVED** [review]

**Comment by nasbench** [comment]
Just a heads up. I closed this pr due to the large number of files that got in here.

I will make a PR with your base64 suggestion and credit you @cyb3rjy0t.

I'll link to this and ping you when it's done. Thanks for your contribution and sorry for the inconvenience

File: rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml
Comment (cyb3rjy0t) [reply rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml]: The threat  explained in the blog "https://thedfirreport.com/2023/01/23/sharefinder-how-threat-actors-discover-file-shares/" can bypass the current detection "[sigma/rules/windows/process_creation/proc_creation_win_susp_sharpview.yml]" as it monitoring the commandline argument "Invoke-Sharefinder" in  Sharpview process.

The "Invoke-Sharefinder" can simply be invoked using powershell process.

**Comment by cyb3rjy0t** [comment]
@nasbench @frack113 I would ask you to revisit the proposed rule as it is monitoring the usage of Invoke-sharefinder module to access file shares. https://thedfirreport.com/2023/01/23/sharefinder-how-threat-actors-discover-file-shares/

I believe, the current rule can be bypassed, if "Invoke-sharefinder" module can be invoked using powershell.

Please let me know if you have any questions

```
detection:
    selection1:
        CommandLine|contains:
            - 'Invoke-ShareFinder' 
    selection2:
        CommandLine|contains:
            # Invoke-ShareFinder Base64
            - 'SW52b2tlLVNoYXJlRmluZGVy'
```

**Comment by cyb3rjy0t** [comment]
@nasbench 

@frack113 

^^^

**Comment by nasbench** [comment]
@cyb3rjy0t I don't understand your inquiry. Your original rule looks for "Invoke-sharefinder" cmdlet or the base64 encoded version. The "Invoke-sharefinder" strings are covered in a process creation rule and a PowerShell script block logging rules. As for the base64, I said I will add it in another PR (today) since that rule needs some tweaks.

Are you suggesting another way? Please explain.

**Pull request closed without merging**
